README.md

```diff
@@ -6,13 +6,27 @@ The recipes folder contains all the recipes in this repo in no particular order.
 
 ## Contributing new recipes
 
+### Principles
+
+#### Only use packages in the latest package set release
+
+In other words, we do not allow anyone to override or add packages in the `packages.dhall` file for the following reasons:
+- Reduces the burden of maintenance for maintainers of this repo
+- Overriding package A to make Recipe X work now might prevent Recipe Y from working (which doesn't need the override)
+- Somewhat reduces the attack surface of malicious recipes (e.g. including a malicious package via addition/override)
+
+Thus, if you want to use a package that's not in the package set, please add it to the package set. Any PRs that use packages not in the package set will not be merged until all of their packages are in the package set.
+
+By implication, recipes that use packages that get dropped from the package set will be moved to a temporary holding folder. If the recipe doesn't get updated within a month, the recipe will be dropped.
```

Comment:
@milesfrain Does this "recipe-dropping" policy seem reasonable to you? I think in times of breaking changes, we would extend this to be longer (e.g. 3 months?)

Reply:
Sure. Do we need a concrete timeline for dropping? Thinking they could just hang out in the broken folder until someone decides to fix them or overwrite the recipe to use an alternative package.

Reply:
Yeah, I think moving them to a 'broken recipes' folder makes more sense. People on later versions can still find them helpful / people trying to update them have less work to do.

Could you merge both this PR and the next one? I got work starting.